lib/cocina/models/validators/description_values_validator.rb

```diff
@@ -64,6 +67,14 @@ def validate_values_for_multiples(hash, path)
           error_paths_multiple << path_to_s(path)
         end
 
+        def validate_title_type(hash, path)
+          # only apply to title.structuredValue with value
+          return unless hash[:value] && path.include?(:title) && path.include?(:structuredValue)
```

Comment:
Wondering if this might match too broadly. Can you check for "title" and "structuredValue" in particular locations in the path?

Reply:
Thanks, I made some updates to this, also as part of checking `relatedResource` titles. There may be some ways cocina gets created that I'm not thinking of, however.